Assets/Scripts/Utilities/RenderSettingsManager.cs

```diff
@@ -46,6 +44,15 @@ void Awake()
 
 	void Update()
 	{
+		dayCycleTimer += Time.deltaTime;
+
+		timeOfDay = (-Mathf.Cos(dayCycleTimer/dayCycleTime * Mathf.Rad2Deg) * 0.5f) + 0.5f;
```

Comment:
Ended up needing to multiply by 2pi, not sure why that wasn't obvious when I was doing it the first time...